Geek-Collectors-Network/geek-collectors-network
# feat(frontend): implement user profile info page

### Summary
Created a user profile information page where they can edit their profile such as username, date of birth, etc. 

### Testing
To verify functionality:

1. Login or sign up to access the /dashboard page.
2. Navigate to the `/profile/edit` page, to review or update user profile information.
